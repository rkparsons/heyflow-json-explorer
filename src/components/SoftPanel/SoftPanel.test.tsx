import { render, screen } from '@testing-library/react';

import { SoftPanel } from '.';

const softPanel = (
    <SoftPanel>
        <button />
    </SoftPanel>
);

test('matches snapshot', () => {
    const { container } = render(softPanel);

    expect(container).toMatchSnapshot();
});

test('renders children', () => {
    render(softPanel);

    const childElement = screen.getByRole('button');

    expect(childElement).toBeInTheDocument();
});
