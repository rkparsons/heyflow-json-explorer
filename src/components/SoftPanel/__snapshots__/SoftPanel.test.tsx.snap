// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<div>
  <div
    class="_root_60b9b3"
  >
    <button />
  </div>
</div>
`;
